components/BottomNavigation/src/MDCBottomNavigationBar.h

```diff
@@ -142,6 +142,11 @@ typedef NS_ENUM(NSInteger, MDCBottomNavigationBarAlignment) {
  */
 @property(nonatomic, assign) CGFloat itemsContentHorizontalMargin;
 
+/**
+ Color of the text represented in the badge
+ */
+@property(nonatomic, strong, nullable) UIColor *badgeTextColor;
```

Comment:
Do we think this should be on `UITabBarItem`? Since users can set colors of each badge uniquely, I suspect that setting the text color for each makes sense as well.